corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -123,7 +123,7 @@ def __call__(self, fn):
 }
 
 
-@register_factory(*INSTANCE_KWARGS_BY_ID.keys())
+@register_factory(*list(INSTANCE_KWARGS_BY_ID.keys()))
```

Comment:
how does this even work (even before this change)? the args would be passed in a non-deterministic order. Is there only ever one arg?

Reply:
it looks like the ```__init__``` for ```register_factory``` could take in the args in any order and ```__call__``` would have the same result.